classes/class-ed-lti.php

```diff
@@ -221,7 +221,7 @@ private function get_username_from_request() {
 	 */
 	private function get_site_data() {
         // phpcs:disable
-		$site_category = isset( $_REQUEST['custom_site_category'] )  ? $_REQUEST['custom_site_category'] :  1;
+		$site_category = isset( $_REQUEST['custom_site_category'] )  ? $_REQUEST['custom_site_category'] :  2;
```

Comment:
I think it would be better to change this from a magic number if possible. Ideally, we should pull this number from the database to ensure it doesn't change. 

This would be my preferred option. If this isn't possible, the second option would be to change this into a constant and have a descriptive name such as `COURSE_SITE_CATEGORY_ID` or something similar. Cheers Richard

Reply:
Ahh, yeah. should have thought about that. Nobody likes a magic number.

Reply:
Thanks for fixing Richard. Looks good!